DentOS_Framework/DentOsTestbed/configuration/README.md

```diff
@@ -34,30 +34,29 @@ Test Bed configuration file format
 }
 ```
 
-Configurations
----------------------
+## Configurations
 
 Below is the directory structure expected for testbed configuration
 
-- configurations/
-  - testbed_config
-    - [testbed_name1]
-      - [dut1]
-        - NTP
-        - QUAGGA_CONFIG
-        - DHCP
-        - QUAGGA_DAEMONS
-        - HOSTNAME
-        - QUAGGA_VTYSH
-        - HOSTS
-        - RESOLV
-        - NETWORK_INTERFACES
-        - SSHD_CONF
-        - KEEPALIVED_CONF
-     - [dut2]
-        - NTP
-        - QUAGGA_CONFIG
-        - ...
-    - [testbed_name2]
-      - [dut1]
-		- ...
+* configurations/
```

Comment:
Just a note for another change, that the bullet usage is different from showing the other directory structures. (Maybe `tree` should be used for all.)

Reply:
i gave it a try looks really nice, i thought it will not work in md files but it does, i'll do a subsequent PR to add that to docs since i found same improvement that can be made in multiple places in docs